source/stylist/__main__.py

```diff
@@ -27,14 +27,29 @@ def parse_cli() -> argparse.Namespace:
     Parse the command line. Returns a dictionary of arguments.
     '''
     description = 'Perform various code style checks on source code.'
+    parsers = [key + ' ' * (8 - len(key)) + '- ' + str(parser)
+               for key, parser in _LANGUAGE_MAP.items()]
+    preproc = [key + ' ' * (8 - len(key)) + '- ' + str(proc)
+               for key, proc in _PREPROCESSOR_MAP.items()]
```

Comment:
You may want to try [str.ljust](https://docs.python.org/3/library/stdtypes.html#str.ljust)?

Reply:
I guess this does not stop a language having multiple pre-processors?

Reply:
A good recommendation, `str.ljust` cleans up the code quite a bit. While I was at it I made the padding dynamic based on the content rather than fixed.

And yes, an extension pipeline can have multiple preprocessors as the pFUnit one does. It has to first go through the pFUnit processor, then the standard Fortran processor.